### DISCUSSION TITLE
[3.x] Form view sidebar control

### DISCUSSION THREAD

**Discussion by kallefrombosnia** [area17/twill#2444] [Ideas]
## Summary

Currently, we don't have much control over the form sidebar. In the last 2.9.0 version, we were introduced to a `getSubmitOptions` method which returns custom action on specific states of the record.

## Describe the solution you'd like

Like `$indexOptions` currently control the index view, I would like to see something like this on the edit forms.
We can control the switch button (draft/ live) per CMS user etc. This also can be implemented with getters and setters in a OO way.

**Comment by ifox** [comment]
@kallefrombosnia I'll move this to a discussion as it isn't clear which feature you're looking to add, and maybe what we added in Twill 3 covered it. Either way, open to discussion, but cleaning up our issues so we can focus our attention on addressing remaining bugs.
